client/src/components/_common/Sidebar/Sidebar.module.css

```diff
@@ -0,0 +1,22 @@
+.root {
+  composes: root from '../../Sidebar/Sidebar.module.scss';
+}
+
+.link {
+  composes: link from '../../Sidebar/Sidebar.module.scss';
+}
+.link--active {
+  composes: link--active from '../../Sidebar/Sidebar.module.scss';
+}
+
+.icon {
+  composes: content from '../../Sidebar/Sidebar.module.scss';
+}
+
+.text {
+  composes: text from '../../Sidebar/Sidebar.module.scss';
+}
+
+.disabled {
+  composes: disabled from '../../Sidebar/Sidebar.module.scss';
+}
```

Comment:
__Joy__
This is super that this works. (Well, except for [`.root`](...).) 

__Question__
Why let main app Sidebar be the source of truth instead of the new common Sidebar?

Reply:
Resolved in comments for another comment.